loop/routes/rooms.js

```diff
@@ -481,4 +481,50 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
       );
     });
   });
+
+  /**
+   * Remove given rooms
+   **/
+  apiRouter.patch('/rooms', auth.requireHawkSession,
+    validators.requireParams('deleteRoomTokens'), function(req, res) {
+      var status = 200;
```

Comment:
Rather than defining the status here, what about doing it at the end with `res.status(status || 200)`.